ViewModel/Saving/PersistenceManagers/AlternativePersistenceManager.cs

```diff
@@ -18,19 +18,21 @@ public class AlternativePersistenceManager : SavingBase, IElementManager
 
         public override Type[] TableColumnTypes => new Type[] { typeof(string), typeof(string) };
 
-
         public AlternativePersistenceManager(Study.FDACache studyCache)
         {
             StudyCacheForSaving = studyCache;
         }
 
-
         public override ChildElement CreateElementFromRowData(object[] rowData)
         {
-            throw new NotImplementedException();
+            string xml = (string)rowData[2];
```

Comment:
Do a size and instance check, or we should make it so rowData is a set of explicitly typed objects instead of an object array.

Reply:
I think turning it into explicitly typed objects would take a long time (isn't this coming from an XML class or something?) and if we wanted to do that, we could add it to task 14 as a backlog fix.

Reply:
I made the "2" a constant and did some checks.